app/instance-initializers/ember-a11y.js

```diff
@@ -16,7 +16,13 @@ Ember.Router.reopen({
   }
 });
 
-export function initialize(application) {
+export function initialize(applicationInstance) {
+  if (arguments.length > 1) {
+    applicationInstance = arguments[1];
+  }
+  const container = applicationInstance.container ? applicationInstance.container : applicationInstance.__container__;
+  const application = container.lookup('application:main');
+
   application._stashedHandlerInfos = stashedHandlerInfos;
```

Comment:
And this could be `instance` instead of `application`. I could be wrong, it has been a long time since 1.13 was a part of my life.